src/datanode/src/instance.rs

```diff
@@ -42,7 +42,7 @@ use table::table::TableIdProviderRef;
 use crate::datanode::{DatanodeOptions, ObjectStoreConfig};
 use crate::error::{
     self, CatalogSnafu, MetaClientInitSnafu, MissingMetasrvOptsSnafu, MissingNodeIdSnafu,
-    NewCatalogSnafu, Result, StartLogStoreSnafu,
+    NewCatalogSnafu, OpenLogStoreSnafu, Result, StartLogStoreSnafu,
```

Comment:
nit-picking: I see both `LogStore` and `LogStore` are used in this file. Do we need to unify them?

Reply:
as well as `log_store` and `logstore`

Reply:
I will fix `Logstore` to `LogStore` in this pr. As to `log_store`, in order to spare crate directory name from `log-store` so I left `logstore` as it is.